custom/enikshay/integrations/nikshay/repeaters.py

```diff
@@ -38,6 +43,73 @@ def allowed_to_forward(self, episode_case):
         )
 
 
+class NikshayFollowupRepeater(CaseRepeater):
+    followup_for_tests = ['end_of_ip', 'end_of_cp', 'posttreatment_6month']
```

Comment:
why did you make this a class variable? Would be good to keep consistent with this and the other values you use in the checks.